src/apis/info/api.ts

```diff
@@ -31,58 +31,72 @@ export class InfoAPI extends JRPCAPI {
   };
 
   /**
-     * Fetches the networkID from the node.
-     *
-     * @returns Returns a Promise<number> of the networkID.
-     */
+   * Fetches the networkID from the node.
+   *
+   * @returns Returns a Promise<number> of the networkID.
+   */
   getNetworkID = async ():Promise<number> => {
     const params:any = {};
     return this.callMethod('info.getNetworkID', params)
       .then((response:RequestResponseData) => response.data.result.networkID);
   };
 
   /**
-   * Fetches the transaction fee from the node.
+   * Fetches the network name this node is running on
    *
-   * @returns Returns a Promise<number> of the transaction fee in nAVAX.
+   * @returns Returns a Promise<string> containing the network name.
    */
-  getTxFee = async ():Promise<BN> => {
-    return this.callMethod('info.getTxFee')
-        .then((response:RequestResponseData) => new BN(response.data.result.txFee));
-  };
-
-  /**
-     * Fetches the network name this node is running on
-     *
-     * @returns Returns a Promise<string> containing the network name.
-     */
   getNetworkName = async ():Promise<string> => this.callMethod('info.getNetworkName')
     .then((response:RequestResponseData) => response.data.result.networkName);
 
   /**
-     * Fetches the nodeID from the node.
-     *
-     * @returns Returns a Promise<string> of the nodeID.
-     */
+   * Fetches the nodeID from the node.
+   *
+   * @returns Returns a Promise<string> of the nodeID.
+   */
   getNodeID = async ():Promise<string> => {
     const params:any = {};
     return this.callMethod('info.getNodeID', params)
       .then((response:RequestResponseData) => response.data.result.nodeID);
   };
 
   /**
-     * Fetches the version of Gecko this node is running
-     *
-     * @returns Returns a Promise<string> containing the version of Gecko.
-     */
+   * Fetches the version of Gecko this node is running
+   *
+   * @returns Returns a Promise<string> containing the version of Gecko.
+   */
   getNodeVersion = async ():Promise<string> => this.callMethod('info.getNodeVersion')
     .then((response:RequestResponseData) => response.data.result.version);
 
   /**
-     * Returns the peers connected to the node.
-     *
-     * @returns Promise for the list of connected peers in <ip>:<port> format.
-     */
+   * Fetches the transaction fee from the node.
+   *
+   * @returns Returns a Promise<BN> of the transaction fee in nAVAX.
+   */
+  getTxFee = async ():Promise<BN> => {
+    return this.callMethod('info.getTxFee')
+        .then((response:RequestResponseData) => new BN(response.data.result.txFee));
+  };
+
+  /**
+   * Check whether a given chain is done bootstrapping
+   * @param chain The ID or alias of a chain.
+   *
+   * @returns Returns a Promise<boolean> of whether the chain has completed bootstrapping.
+   */
+  isBootstrapped = async (chain:string):Promise<boolean> => {
```

Comment:
Should we default to the X Chain?

Reply:
What does Gecko do? Is this a required parameter in Gecko? I do not see a default and it doesn't say it's optional. I want to get in the habit of being more strict because someone could forget this param and start getting the wrong chainID and while not likely catastrophic, it could be hard to understand on their end.

Reply:
@cgcardona confirmed it's required, so we'll keep it as it stands.